### ISSUE TITLE
add radar name as column to vpi data.frame

### ISSUE THREAD

**Issue by adokter** [adokter/bioRad#425]
Suggestion by @bmvandoren: Currently vpi objects (which are data.frame objects) do not have the radar name as a column. 

For vpts objects we currently have `as.data.frame(vpts)` adding a column with the radar name. We could do the same for vpi, so have `as.data.frame(vpi)` add a column with the radar name, or change the structure of vpi objects (which is simply a data.frame) and add radar as column. @peterdesmet any preference for those two options?

**Comment by peterdesmet** [comment]
I would make this a property of `vpi`. A `vpi` being a dataframe, I think it will get confusing if `vpi` (without radar) looks different from `as.data.frame(vpi)` (with radar). In addition, we're not really advertising the `as.data.frame(vpi)` function, so I wouldn't hide secret functionality in that one.

**Comment by bmvandoren** [comment]
The output of integrate_profile() is already a data frame, and it doesn’t have a radar column - this is where I’m suggesting adding that. Calling as.data.frame on on the output doesn’t change anything.

**Comment by peterdesmet** [comment]
Yes, in the current implementation the output of `integrate_profile()` and calling `as.data.frame()` on it looks the same: the first one just has an additional class `vpi`. I would keep that behaviour and add the radar column to the output of `integrate_profile()`.

```r
> vpi <- integrate_profile(example_vpts)
> class(vpi)
[1] "vpi"        "data.frame"
> vpi_df <- as.data.frame(vpi)
> class(vpi_df)
[1] "data.frame"
```
